<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyKey by G4</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>731</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="5" rowspan="2" colspan="2">
      <widget class="QPushButton" name="pb_Select">
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="2">
      <widget class="QPushButton" name="pb_Clear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4" rowspan="2">
      <widget class="QPushButton" name="pb_Eject">
       <property name="text">
        <string>Eject selected</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="3">
      <widget class="QListWidget" name="usbDrives"/>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Create new folder(s)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QRadioButton" name="radio_folder_yes">
       <property name="text">
        <string>yes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item row="4" column="5" colspan="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Erase selected</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QRadioButton" name="radio_folder_no">
       <property name="text">
        <string>no</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pb_browse">
       <property name="text">
        <string>Browse files</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4" colspan="3">
      <widget class="QLineEdit" name="new_folder">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pb_browse_folders">
       <property name="text">
        <string>Browse folders</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLineEdit" name="address_mail"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About the fantastic team</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
